### PR TITLE
matched id between hero and announcement detail

### PR DESCRIPTION
**Pull request by jjaykim**

### CODE REVIEW

File: client/src/components/hero/HeroContentDesktop.tsx [hunk 1]
```diff
@@ -39,6 +41,10 @@ const UnstyledHeroContentDesktop: FunctionComponent<HomeDesktopProps> = ({
     return 15;
   }, [isMedium]);
 
+  const announcementID = (id: string) => {
```
Comment (dhmoon91) [inline client/src/components/hero/HeroContentDesktop.tsx:44]: rename the function that represents what this does.
This sounds like variable.

File: server/graphql/schema/queries/getHomeViewer.ts [hunk 2]
```diff
@@ -29,9 +29,7 @@ export const getHomeViewer = {
     }
 
     const announcements = await Announcement.findAll({
-      where: { showOnHomepage: true },
       order: [['createdAt', 'DESC']],
-      limit: 5,
```
Comment (dhmoon91) [inline server/graphql/schema/queries/getHomeViewer.ts:34]: i don't like this.. we make separate query on anouncement page so we will be running two same queries that returns same data..

Comment (jjaykim) [reply server/graphql/schema/queries/getHomeViewer.ts:34]: history.push 할 때 `league detail` id 랑 맞추려면 전체 `announcement length` 가 필요해서 이렇게 햇는데,, 다른 방법을 생각해보겟습니다ㅠㅠ

File: client/src/components/hero/HeroContentDesktop.tsx (hunk 1 above)
Comment (jjaykim) [reply client/src/components/hero/HeroContentDesktop.tsx:44]: Fixed!

File: server/graphql/schema/queries/getHomeViewer.ts (hunk 2 above)
Comment (jjaykim) [reply server/graphql/schema/queries/getHomeViewer.ts]: fixed!

**Review by dhmoon91 — APPROVED** [review]

**Pull request merged**